modules/userId/eids.js

```diff
@@ -1,4 +1,5 @@
 import * as utils from '../../src/utils.js';
+import {isArray} from "../../src/utils";
```

Comment:
utils.js is already imported above. Can you remove this second declaration and use the utils var instead?